packages/extension/src/providers/polkadot/networks/interlay/libs/assetinfo-orml.ts

```diff
@@ -0,0 +1,160 @@
+import API from "@/providers/polkadot/libs/api";
+import { SubstrateNetwork } from "@/providers/polkadot/types/substrate-network";
+import { hexToString, hexToBn } from "@polkadot/util";
+import {
+  InterlayOrmlAsset,
+  InterlayOrmlAssetOptions,
+  OrmlAssetType,
+} from "../types/interlay-orml-asset";
+import { OrmlTokensAccountData } from "../../acala/types/acala-orml-asset";
+import { toBN } from "web3-utils";
+import { KnownTokenDisplay } from "@/providers/polkadot/types";
+import { SubstrateNativeToken } from "@/providers/polkadot/types/substrate-native-token";
+
+type AssetMetadata = {
+  name: `0x${string}`;
+  symbol: `0x${string}`;
+  decimals: number;
+  minimalBalance: number | `0x${string}`;
+};
+
+type NativeAsset = Record<string, string>;
```

Comment:
These assets are for Acala I think. For us it would be something like this (not tested), see also https://docs.interlay.io/#/developers/assets:


```ts
type Token = string;
type ForeignAsset = number;
```

Not sure if we can just ignore the more complex LendToken, LpToken, and StableLpToken or if that would result in an error.